addons/web/controllers/report.py

```diff
@@ -133,7 +133,10 @@ def report_download(self, data, context=None, token=None):  # pylint: disable=un
             else:
                 return
         except Exception as e:
-            _logger.exception("Error while generating report %s", reportname)
+            if reportname.startswith('studio_customization'):
+                _logger.warning("Error while generating report %s", reportname, exc_info=True)
+            else:
+                _logger.exception("Error while generating report %s", reportname)
```

Comment:
I mean no need to check if it's a studio customization
We don't want something linked with sentry in our codebase
```suggestion
           _logger.warning("Error while generating report %s", reportname, exc_info=True)
```

Reply:
Hello @abz-odoo 
Changes have been done, Please check it.
Thanks :sweat_smile: